src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  --main-color: #00acdc;
+
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  margin: auto 0;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.nav {
+  margin: auto 50px;
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin: auto 0;
+  padding: 0;
+  height: 100%;
+}
+
+.nav__list__item {
+  list-style: none;
+
+}
+
+.nav__list__link {
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  height: 60px;
+  line-height: 60px;
+}
+
+.logo {
+  margin: auto 50px ;
+}
+
+.logo__link {
+  text-align: center;
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav__list__link:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__list__item:hover {
+  color: var(--main-color);
+}
+
+.is-active {
+  color: var(--main-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  height: 4px;
+  width: 37px;
+  border-radius: 8px;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: 0;
```

Comment:
let's improve your code. Please check this article to write the correct CSS properties order https://netology-university.bitbucket.io/codestyle/css/#order